cmd/ko/main.go

```diff
@@ -36,3 +35,4 @@ func main() {
 		log.Fatalf("error during command execution: %v", err)
 	}
 }
+
```

Comment:
This is failing [in travis](https://travis-ci.org/google/ko/jobs/591790322) because it doesn't match [`go fmt`](https://blog.golang.org/go-fmt-your-code).